packages/@cagov/accordion/src/index.js

```diff
@@ -1,21 +1,33 @@
 export class CaGovAccordion extends window.HTMLElement {
-  connectedCallback () {
+  connectedCallback() {
     this.classList.add('prog-enhanced');
     this.expandTarget = this.querySelector('.card-container');
     this.expandButton = this.querySelector('.card-header');
     this.expandButton.addEventListener('click', this.listen.bind(this));
     this.activateButton = this.querySelector('.card-header');
     this.eventType = this.dataset.eventType ? this.dataset.eventType : 'click';
+    this.addStyle(this); // adding local styles
```

Comment:
an improvement would be to only add the style to the document once instead of each time the accordion is instantiated. we can address that in a subsequent ticket too